src/core/manager.c

```diff
@@ -949,10 +949,14 @@ int manager_new(LookupScope scope, ManagerTestRunFlags test_run_flags, Manager *
                         return r;
         }
 
-        m->taint_usr =
+        m->taint_split_usr =
                 !in_initrd() &&
                 dir_is_empty("/usr") > 0;
 
+        m->taint_unmerged_usr =
+                !in_initrd() &&
+                is_symlink("/bin") == 0;
```

Comment:
i'd probably generate a LOG_DEBUG message in case this returns < 0.

Reply:
refactored to do the same as the /var/run check with readlink_malloc